File: prog/c/discountt.c
#include <stdio.h>


int main(void) 
{
	
	double quantity,price,amount,discount;

	printf("Enter quantity and price:");	
	scanf("%lf %lf",&quantity, &price);

	amount=quantity*price;

	if(amount>5000)

	{

		discount=amount*0.10;


	}
	
	else if(amount > 1000)
	{
		discount=amount*0.05;

	}
	else
	{
		discount=0;
	}
	amount = amount - discount;
	printf("%lf",amount);
	
	return 0;

}

File: prog/python/discountt.py
noi=int(input("Enter the no of Items:"))
price=float(input("Enter the price:"))
amount=noi * price
discount=0
if(amount>=5000 and amount<10000):
    discount=amount*0.05
elif(amount>=10000):
    discount=amount*0.1

amount=amount-discount

print("Bill Amount: {}".format(amount))

